apps/site-landing/src/containers/HelpDocs/messages.ts

```diff
@@ -10,14 +10,6 @@ export const scope = 'app.components.Header';
 export default defineMessages({
   header: {
     id: `${scope}.header`,
-    defaultMessage: 'What would you like to know?',
-  },
-  subHeader: {
-    id: `${scope}.subHeader`,
-    defaultMessage: 'See what interests you from the menu on the left',
-  },
-  blurb: {
-    id: `${scope}.blurb`,
-    defaultMessage: 'Or, for a quick overview of our company try the below items',
+    defaultMessage: 'FAQ',
```

Comment:
feel free to move these messages inside the component file, and delete this `messages.ts` file.  The rest of our code embeds the messages directly where they are used